src/modules/Channel/components/SuggestedMentionList/index.tsx

```diff
@@ -129,7 +123,18 @@ function SuggestedMentionList(props: SuggestedMentionListProps): JSX.Element {
           logger.error('SuggestedMentionList: Fetching member list failed', error);
         }
       });
-  }, [channelInstance?.url, searchString]);
+  }, [
+    channelInstance?.url,
+    // We have to be specific like this or React would not recognize the changes in instances.
+    channelInstance.members.map((member: Member) => member.userId).join(),
```

Comment:
Since user ids are not expected to change within the same member list, so it seems like this line can be removed.

Reply:
that was astute. i remove it!